docs/releasenotes/10.3.0.rst

```diff
@@ -85,3 +90,8 @@ Release GIL when fetching WebP frames
 
 Python's Global Interpreter Lock is now released when fetching WebP frames from
 the libwebp decoder.
+
+Add release notes for 2.3.1, 2.3.2, 2.5.2
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
```

Comment:
Apologies if I've missed something, but this doesn't need to be in the 10.3.0 release notes, right?

Reply:
If you consider the addition of any security information about Pillow to the documentation to be relevant… and you consider that the information was added during the 10.3.0 development cycle and will be included in the 10.3.0 release then… maybe? I'm not sure how else you'd draw attention to the changes if not to make an entry in the release notes. But if you feel strongly that it should not be in the release notes for 10.3.0 you are welcome to remove it, because I don't feel that strongly about it 😄

Reply:
No strong feelings, I thought it might be a copy-paste error or something.
I just imagine that anyone reading the 10.3.0 release notes has long since forgotten about the existence of Pillow 2.5.2.

Reply:
Yeah, feels a bit meta to document addition of documentation.

And these releases are from 2014, for Python 2.6 - 3.3/3.4, no need to call them out now.

Reply:
`raise Subjective` again. This effort is intended to provide a comprehensive look at CVE data from 1995 to 2024 during which time we have 3 major periods:

- 1995-2010: No CVEs that I'm aware of.
- 2010-2019: A few CVEs, requested by @wiredfool and/or 3rd party that reported the issue.
- 2019-2024: Many CVEs, requested by Tidelift and/or 3rd party that reported the issue.

I'm glad you asked, because I will replace the `TODO` with this information in some form… because prior to now I wasn't sure what to write 😄

Reply:
@hugovk Again, if you feel that strongly about it please remove, I don't have a strong preference.

Reply:
Thanks, will do!